cairo_programs/secp_integration_test.cairo

```diff
@@ -0,0 +1,120 @@
+%builtins range_check
+
+from starkware.cairo.common.cairo_secp.bigint import BigInt3, bigint_mul, nondet_bigint3, bigint_to_uint256 
+from starkware.cairo.common.cairo_secp.signature import validate_signature_entry
+from starkware.cairo.common.cairo_secp.constants import N0, N1, N2, BASE
+from starkware.cairo.common.cairo_secp.ec import EcPoint, ec_add, ec_mul, ec_negate, ec_double, fast_ec_add, compute_doubling_slope, compute_slope
+from starkware.cairo.common.math import assert_nn_le, assert_not_zero
+from starkware.cairo.common.uint256 import Uint256
+
+# Computes x * s^(-1) modulo the size of the elliptic curve (N).
+func mul_s_inv{range_check_ptr}(a : BigInt3, b : BigInt3) -> (res : BigInt3):
+    %{
```

Comment:
I think it would be better if we call the function that uses this hint instead of writing it by hand

Reply:
Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/370/commits/fb18c767e4f3d7d84f88469bdcc61ea4474808be